kivy/core/window/_window_sdl2.pyx

```diff
@@ -419,6 +419,14 @@ cdef class _WindowSDL2Storage:
     def flip(self):
         SDL_GL_SwapWindow(self.win)
 
+    def set_window_opacity(self, opacity):
+        return SDL_SetWindowOpacity(self.win, opacity)
```

Comment:
There's no need for a return in this function and it might even return garbage or crash the app with SDL2 returns such as [this](https://hg.libsdl.org/SDL/file/b6315b6bc32f/src/video/SDL_video.c#l2264).